.github/labels.md

```diff
@@ -0,0 +1,252 @@
+
+??? Requests don't belong into the issue tracker IMO.
+
+#### Label `category:tweak`
+
+The PR contains insignificant code changes that don't involve features,
+refactoring or bug fixing but code style grooming or value tweaking.
+
+#### Label `component:AI`
+
+The Issue/PR deals with the python AI decision making code or affects it.
+
+#### Label `component:art music text assets`
+
+The Issue/PR deals with creating or modifying game assets or verification if the
+assets match the style goals of the game.
+
+#### Label `component:build system`
+
+The Issue/PR deals with CMake, Visual Studio, Xcode, the respective platform
+SDKs or the general build process.
+
+#### Label `component:deployment`
+
+The Issue/PR deals with the deployment process or the software used create the
+installer or other distribution packages.
+
+#### Label `component:documentation`
+
+The Issue/PR deals with Doxygen or primarily contains documentation.
+
+#### Label `component:game mechanic`
+
+The Issue/PR deals with the currently used or planned game mechanics.  The
+content of the PR may require extensive gameplay balance testing.
+
+#### Label `component:infrastructure`
+
+The Issue/PR deals with non-game related infrastructure that aids development.
+For example this includes issues with the project homepage, used external
+services like continuous integration or hosting of release binaries.
+
+#### Label `component:internal`
+
+The Issue/PR deals with any project component that has no explicit `component`
+label.
+
+#### Label `component:network`
+
+The Issue/PR deals with TCP/IP networking and establishing a test network
+infrastructure.
+
+#### Label `component:scripting`
+
+The Issue/PR deals with the FOCS language and probably game mechanics.  Probably
+most used in conjunction with `component:game mechanic` and
+`component:art music text assets`.
+
+#### Label `component:translation`
+
+The Issue/PR deals with the game style, game terminology, stringtable syntax
+and the translation process in general.  It maybe helpful to know the language
+of the translation to proofread.
+
+#### Label `component:UI`
+
+The Issue/PR deals with the game interface, the expected user experience and
+graphical APIs used (GiGi and OpenGL).
+
+#### Label `status:cherry-pick for release`
+
+The PR should be applied to the currently open release branch.  Use this label
+sparsely, only for bug fixes, and document a minimal set of commits that should
+be considered by the release manager for cherry-picking.  Cherry-picking
+PRs onto release branches should fix bugs, not introduce new ones.
+
+#### Label `status:duplicate`
+
+The topic of the Issue/PR is or was already addressed by another Issue/PR.  Add
+the number of the addressing Issue/PR in a comment and close this Issue/PR.  For
+PRs document why the other PR was chosen of this PR.  If the closed Issue
+contains new knowledge that is valuable to solve the addressing Issue copy
+it over to the addressing Issue.
```

Comment:
Shouldn't that be now (after dropping the use of "Issue" for both issues and PRs): "the closed Issue/PR" and "addressing PR"? (Because only PRs *address* something.)